README.md

```diff
@@ -94,11 +95,12 @@ end
 
 Our branches (in order or stability are):
 
-| Branch      | Environment | AWS Account     |
-|:------------|:------------|:----------------|
-| master      | none        | none            |
-| development | development | aws-sandbox     |
-| production  | production  | aws-digital-dev |
+| Branch      | Environment | AWS Account      |
+|:------------|:------------|:-----------------|
+| master      | none        | none             |
+| development | development | nypl-sandbox     |
+| qa          | qa          | nypl-digital-dev |
+| production  | production  | nypl-digital-dev |
```

Comment:
Is the use of a `production` branch (instead of just using `master` for the Prod environment) the thing that Kevin was commenting on as being different from standard practice in one of the old PRs?

Reply:
Yeah - some of our apps use a `production` branch...others use `master` as the stable/production branch.
The split is mostly between our Ruby and JS apps. I think the idiom in JS development is to use `master` as stable and in many Ruby libraries `master` is bleeding edge....

We haven't settled on which we want to try to commit to entirely - so for now the approach is to document which approach the repo uses in its README.